apps/vth-frontend/src/query/index.ts

```diff
@@ -12,3 +12,15 @@ query getNotFoundPage($locale: I18NLocaleCode){
   }
 }
 `);
+
+export const GET_HOME_PAGE = gql(`
+query getHomePage {
+  homepage {
+    data {
+      attributes {
+        Title
```

Comment:
Just a quick suggestion: Could we use lowercase for the field names instead of starting with uppercase?
Using lowercase for field names helps maintain consistency and is a common convention in GraphQL. 
Thanks!

Reply:
Thanks, didn't realise till just now we can customize the display string so the admin fields are still dutch and correctly cased.